plugin/trino-druid/src/test/java/io/trino/plugin/druid/TestingDruidServer.java

```diff
@@ -238,13 +242,26 @@ void ingestData(String datasource, String indexTaskFile, String dataFilePath)
         middleManager.withCopyFileToContainer(forHostPath(dataFilePath),
                 getMiddleManagerContainerPathForDataFile(dataFilePath));
         String indexTask = Resources.toString(getResource(indexTaskFile), Charset.defaultCharset());
+        ingest(indexTask, datasource);
+    }
+
+    void ingestDataWithoutTaskFile(String indexTask, String dataFilePath, String datasource)
+            throws IOException, InterruptedException
+    {
+        middleManager.withCopyFileToContainer(forHostPath(dataFilePath),
+                getMiddleManagerContainerPathForDataFile(dataFilePath));
+        ingest(indexTask, datasource);
+    }
 
+    void ingest(String indexTask, String datasource) throws IOException, InterruptedException {
         Request.Builder requestBuilder = new Request.Builder();
         requestBuilder.addHeader("content-type", "application/json;charset=utf-8")
                 .url("http://localhost:" + getCoordinatorOverlordPort() + "/druid/indexer/v1/task")
                 .post(RequestBody.create(null, indexTask));
         Request ingestionRequest = requestBuilder.build();
-        try (Response ignored = httpClient.newCall(ingestionRequest).execute()) {
+
+        try (Response taskIdResponse = httpClient.newCall(ingestionRequest).execute()) {
+            log.info(taskIdResponse.body().string());
```

Comment:
Is this required only for debugging? If so, change the log level or remove.

Reply:
When running the connector tests this is also like a progress bar that tells what's going on instead of nothing showed.
Or maybe I can use DEBUG level.

Reply:
Then let's log some informational message instead of the entire HTTP response body. Agreed about the progress indication.

Reply:
OK but actually the response body is just `{"taskId": "xxx"}`